packages/ui/src/app/pages/Validators/ValidatorList.stories.tsx

```diff
@@ -113,4 +142,44 @@ export default {
 
 type Story = StoryObj<typeof ValidatorList>
 
-export const Statistics: Story = {}
+export const StatisticsAndLists: Story = {}
+
+export const TestsFilters: Story = {
+  play: async ({ canvasElement, step }) => {
+    const screen = within(canvasElement)
+
+    const searchElement = screen.getByPlaceholderText('Search')
+    const verificationFilter = screen.getAllByText('Verification')[0]
+    const stateFilter = screen.getAllByText('State')[0]
+    const clearFilter = () => {
+      userEvent.click(screen.getByText('Clear all filters'))
+    }
```

Comment:
This should be awaited even though `userEvent.click` doesn't return a promise (my guess is that it's because storybook monkeypatches these methods when they're called in the storybook ui).

Also IMO it would be better to save `screen.getByText('Clear all filters')` in a constant (just like how you did it for `searchElement`, `verificationFilter`, and `stateFilter`. And then call  `await userEvent.click(clearFilter)` every time the tests need to clear the filters.

Reply:
I fixed the code according to your review. `screen.getByText('Clear all filters')` occurs error when I define it as a constant, because it does not appear when all validators are shown.